rpc/js/src/peer.ts

```diff
@@ -3,9 +3,29 @@ interface ReadyPeer {
   dc: RTCDataChannel;
 }
 
+export function addCustomSdpFields(
+  sdpFields?: object,
+  localDescription?: RTCSessionDescription | null
+) {
+  let description = {
+    sdp: localDescription?.sdp,
+    type: localDescription?.type,
+  };
+  if (sdpFields) {
+    Object.keys(sdpFields).forEach((key) => {
+      description.sdp = [
+        description.sdp,
+        `a=x-${key}:${sdpFields[key as keyof typeof sdpFields]}\r\n`,
```

Comment:
i don't think the x- should be automatically added

Reply:
If this is for custom sdp values, the `x-` is a standard to add in, according to https://www.ietf.org/rfc/rfc2327.txt.
> Unregistered attributes should begin with "X-" to
   prevent inadvertent collision with registered attributes.  In either
   case, if an attribute is received that is not understood, it should
   simply be ignored by the receiver.

Reply:
ya but users of the utility may want to add attributes that are registered

Reply:
Ah, I thought we didn't want them to be able to change attributes until further notice. If that's the case, I will change the function name as well as get rid of the `x-`.